lib/requests.js

```diff
@@ -149,6 +158,12 @@ QueryRequest.prototype.write = function (encoder) {
   //v3: <query>
   //      <consistency><flags>[<n>[name_1]<value_1>...[name_n]<value_n>][<result_page_size>][<paging_state>][<serial_consistency>][<timestamp>]
   var frameWriter = new FrameWriter(types.opcodes.query);
+  var headerFlags = this.options.traceQuery ? types.frameFlags.tracing : 0;
```

Comment:
Makes sense :+1:.  I think supporting custom payloads on the query itself is sufficient for now.  I logged [NODEJS-134](https://datastax-oss.atlassian.net/browse/NODEJS-134) for future support.